### PR TITLE
made sure that followed pool.csv will be created if not present

### PR DESCRIPTION
**Pull request by timgrossmann**
Fixes 
```
ERROR - set_automated_followed_pool error [Errno 2] No such file or directory: './logs/username_followedPool.csv'
unable to get followers and following information
```

### CODE REVIEW

**Comment by timgrossmann** [comment]
Could somebody please doublecheck this?

**Comment by dvlprlife** [comment]
I processed a few accounts with an empty log folder and didn't have any issues; file was created and used. I have also worked with the followRestriction.json being absent in #1150  / Fixes #1149

File: instapy/unfollow_util.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 def set_automated_followed_pool(username, logger):
     automatedFollowedPool = []
     try:
-        with open('./logs/' + username + '_followedPool.csv') as \
+        with open('./logs/{username}_followedPool.csv'.format(username=username), 'r+') as \
```
Comment (CharlesCCC) [inline instapy/unfollow_util.py]: Can you replace the line with this ? 
`'./logs/{}_followedPool.csv'.format(username)`

**Comment by RoyBkker** [comment]
Still have this error on Windows, Python 3.6.3. Chrome 63 and latest chrome driver..

**Pull request merged**